src/runtime_src/core/tools/xbmgmt2/xbmgmt-bash-completion

```diff
@@ -45,6 +45,10 @@ _command_word_xbmgmt_completion()
       ;;
     --image)
       ;;
+    --input)
```

Comment:
This sub-cmd is only for xbmgmt so I don't think this applies here

Reply:
This is the completion script for xbmgmt and there is another script for xbutil. I am not sure what issue you see here, do you mind clarifying?

Reply:
My bad, I misread this script name as xbutil. This is good